projects/sbb-esta/angular-maps/esri-web-map/src/esri-types/esri-types.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@angular/core';
+
+import { EsriLoaderService } from '../../../esri-config/src/esri-loader.service';
```

Comment:
When importing from another submodule, use `from '@sbb-esta/angular-maps/esri-config';` (or `from '@sbb-esta/angular-maps/name-of-submodule';`